modules/heat_conduction/src/kernels/HeatConductionTimeDerivative.C

```diff
@@ -10,68 +10,39 @@ template<>
 InputParameters validParams<HeatConductionTimeDerivative>()
 {
   InputParameters params = validParams<TimeDerivative>();
+  params.addClassDescription("Time derivative term $\\rho c_p \\frac{\\partial T}{\\partial t}$ of "
+                             "the heat equation for quasi-constant specific heat $c_p$ and the density $\\rho$.");
+
   // Density may be changing with deformation, so we must integrate
   // over current volume by setting the use_displaced_mesh flag.
   params.set<bool>("use_displaced_mesh") = true;
-  params.addParam<bool>("use_heat_capacity",
-                        false,
-                        "Use a single material property, 'heat_capacity', "
-                        "as the coefficient of the time derivative.");
 
-  params.addParam<MaterialPropertyName>("specific_heat_name",
+  params.addParam<MaterialPropertyName>("specific_heat", //_name
```

Comment:
We couldn't find any examples of any models using anything other than the default values in the herd app repositories we looked in, so we doubt that this will break anyone's models.